src/test/java/com/google/api/codegen/testdata/discogapic/java/java_simplecompute.v1.json.baseline

```diff
@@ -2365,88 +2590,352 @@ public final class Error implements ApiMessage {
     return null;
   }
 
-  public List<Errors> getErrorsList() {
-    return errors;
+  public Float getFloatie() {
+    return floatie;
+  }
+
+  public String getName() {
+    return name;
+  }
+
+  public Double getPrecisionFloatie() {
+    return precisionFloatie;
+  }
+
+  public Address getAddress() {
+    return primaryAddress;
+  }
+
+  public Address getAddress() {
```

Comment:
This duplicate naming issue will be fixed in https://github.com/googleapis/toolkit/pull/1892.